holidays/countries/chile.py

```diff
@@ -237,7 +239,7 @@ def _populate_bank_holidays(self):
         if 1957 <= self._year <= 1975:
             self._add_holiday_jun_30(name)
 
-        if self._year >= 1956 and self._year != 1997:
+        if (1956 <= self._year <= 2024) and (self._year != 1997):
```

Comment:
You might want to update the `test_bank_holidays`] in  [`test_chile.py`](https://github.com/vacanza/holidays/blob/dev/tests/countries/test_chile.py#L277) file as well